tests/test_legislations.py

```diff
@@ -13,54 +13,8 @@
 
 
 def check_legislation_xml_file(year):
-    legislation_tree = conv.check(legislationsxml.make_xml_legislation_info_list_to_xml_element(False))(
-        tax_benefit_system.legislation_xml_info_list, state = conv.default_state)
-    legislation_xml_json = conv.check(legislationsxml.xml_legislation_to_json)(
-        legislation_tree,
-        state = conv.default_state,
-        )
+    compact_legislation = tax_benefit_system.get_legislation(year)
```

Comment:
What are we testing exactly ? @michelbl do you think it is relevant ?
I don't know much this part of OpenFisca.